Resources/Textures/_NF/Structures/Machines/shredder.rsi/meta.json

```diff
@@ -0,0 +1,41 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "license": "CC-BY-NC-SA-3.0",
+  "copyright": "Created by [Data-Redacted] https://github.com/IData-RedactedI/frontier-station-14 ",
+  "states": [
+    [
```

Comment:
somthing is upsetting the validator here. Is this bracket here necessary? it appears this is all wrapped up in a double [[]] instead of a single one maybe?

Reply:
Yeah thats a surplus [] there, whoops